knowledge_doc_converter/.env.example

```
@@ -27,6 +27,11 @@ MINERU_FORMULA_ENABLE=true
 MINERU_TABLE_ENABLE=true
 MINERU_POLL_INTERVAL_SECONDS=3
 MINERU_MAX_WAIT_SECONDS=3000
+MINERU_SUBMIT_TIMEOUT_SECONDS=60
+MINERU_STATUS_TIMEOUT_SECONDS=30
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINERU_STATUS_TIMEOUT_SECONDS key should go before the MINERU_SUBMIT_TIMEOUT_SECONDS key

(UnorderedKey)